macros/fivetran/base/fivetran_fb_ad_creatives.sql

```diff
@@ -0,0 +1,46 @@
+{% macro fivetran_fb_ad_creatives() %}
+
+    {{ adapter_macro('facebook_ads.fivetran_fb_ad_creatives') }}
+
+{% endmacro %}
+
+
+{% macro default__fivetran_fb_ad_creatives() %}
```

Comment:
if you use dbt utils for the url parsing then you don't need two different versions of this model.